interpreter/syntax/types.ml

```diff
@@ -10,11 +10,13 @@ type 'a limits = {min : 'a; max : 'a option}
 type mutability = Immutable | Mutable
 type table_type = TableType of Int32.t limits * ref_type
 type memory_type = MemoryType of Int32.t limits
+type event_type = EventType of int32
```

Comment:
For consistency, can you either change the other occurrences to `int32` as well?

Reply:
Done